src/sqlcode.l

```diff
@@ -1,10 +1,10 @@
 /******************************************************************************
  *
- * Copyright (C) 1997-2014 by Dimitri van Heesch.
+ * Copyright (C) 1997-2020 by Dimitri van Heesch.
```

Comment:
What is the relevance of the changes in this file in respect to the hyphenation in tables?

Reply:
@deem0n I think you need to pull in the externally made changes in the master branch, then switch to your feature branch and to a `git rebase master` before updating the pull request (probably you did a merge now?). Otherwise changes made by others will appear inside your pull request!

Reply:
Yes, it seems I messed up with git. Is it ok if I close this PR and make another one with just one file chaned (src/latexdocvisitor.cpp) ?

Reply:
@deem0n sure no problem.

Reply:
Please see #8030